services/frontend/src/AppRouter.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import {Switch, Route} from "react-router";
+import {Profile, People} from "scenes/";
+import ScreenUrls from "utils/screenUrls";
+import { Navbar } from "components/";
+
+const AppRouter = () => (
+    <div id="app">
+        <Navbar />
+        <Switch>
+            <Route path={ScreenUrls.PROFILE} component={Profile} />
+            <Route path={ScreenUrls.PEOPLE} component={People} />
+            <Route path={ScreenUrls.PROFILE} component={Profile} />
```

Comment:
I changed it to `ScreenUrl.PROJECTS` it just looks slightly different on the live page since there's no scene setup for it yet so its defaulting to another route.